sentence-transformer/Makefile

```diff
@@ -0,0 +1,24 @@
+ROOT=$(shell dirname ${PWD})
+VENV=.venv
+$(VENV):
+	$(MAKE) $(VENV)-update
+
+.PHONY: $(VENV)-update
+$(VENV)-update: virtualenv-installed
```

Comment:
I want to (try to) start transitioning off of virtualenv (which requires an extra pip install) to the `venv` tool that comes baked into python 3. Can you take a stab at using venv here?